src/main/java/ru/testtask/service/ProjectService.java

```diff
@@ -58,5 +64,23 @@ public void updateProject(Project project) {
         return projectRepo.findAll();
     }
 
+    public boolean isOwner(String id){
```

Comment:
I suggested the wrong method name. Let's rename it to `isCurrentUserOwnerOf(String projectId)`